C_CPP/Fundamentals/Looping Control Structure or Statements/Looping Control Structure.md

```diff
@@ -0,0 +1,164 @@
+<div align="center">
+<h1>Looping Control Structure in C/C++</h1>
+</div>
+
+## Loop/Iteration: The repeated execution of a statement or group of statements until a given condition is false or dissatisfied.
+
+## Types of Loops in C/C++
+
+1. **for loop** (Counter loop/single statement loop)
+2. **while loop** (Entry-controlled/Pre-conditional)
+3. **do while loop** (Entry-controlled/Post-conditional)
```

Comment:
Do-while loop is exit controlled. Proof-read your content before submitting.

Reply:
sorry, I'll correct this.